platform/admin/core/admin-core-impl/src/test/java/org/codice/ddf/admin/core/impl/AdminAlertImplTest.java

```diff
@@ -111,7 +111,7 @@ public void testGetAlertsWithSingleDetail() throws Exception {
   public void testGetAlertsPersistentStoreError() throws Exception {
     when(persistentStore.get(eq("alerts"), any())).thenThrow(new PersistenceException("error"));
     List<Map<String, Object>> alertList = adminAlert.getAlerts();
-    assertThat(alertList.size(), equalTo(0));
+    assertThat(alertList.size(), equalTo(1));
```

Comment:
Could also assert that it is the solr alert here.

Reply:
I added the assertion that it is the solr alert